### DISCUSSION TITLE
I want to fold the content of the annotation in VSCode Editor. how to do with it?

### DISCUSSION THREAD

**Discussion by liaodalin19903** [microsoft/vscode-discussions#356] [Extension Development QnA] [answered]
![image](https://user-images.githubusercontent.com/12727582/210071921-6cf86908-5dae-44f5-a1d8-1ce6e1180686.png)

I want to fold the content of the annotation in VSCode Editor. 

but I didn't found the API of implementing my requirement. 

whether VSCode API support it?

**Comment by burkeholland** [comment, marked as answer]
I think what you're looking for is the FoldingRangeProvider. Check out the following code as an example... https://github.com/zokugun/vscode-explicit-folding/blob/master/src/folding-provider.ts
